%YAML:1.0
names:
   - Muesli
   - Spoon
   - Milk
   - Bowl
   - Mug
Muesli:
   category: Muesli
   color: [green,Green]
   shape: [box,Cubic]
   material: [Cartoon]
   openability: true
   folderPath: none
   filePath: pcd_models/KoellnMuesliKnusperHonigNuss/KoellnMuesliKnusperHonigNuss.pcd
   fromPackage: true
   packageName: rs_resources
   scale: 100.0
   axisMap: [3,1,2]
Spoon:
   category: Spoon
   color: [white, White, black, Black, gray, Gray]
   shape: [flat, Filiform]
   material: [Plastic]
   openability: false
   folderPath: none
   filePath: pcd_models/SoupSpoon/SoupSpoon.pcd
   fromPackage: true
   packageName: rs_resources
   scale: 100.0
   axisMap: [1,-3,2]
Milk:
   category: Milk
   color: [blue, white,Blue, White]
   shape: [box,Cylindrical]
   material: [Cartoon]
   openability: true
   folderPath: none
   filePath: pcd_models/WeideMilchSmall/WeideMilchSmall.pcd
   fromPackage: true
   packageName: rs_resources
   scale: 100.0
   axisMap: [1,-3,2]
Bowl:
   category: Bowl
   color: [red,Red]
   shape: [round,Conical]
   material: [Plastic]
   openability: false
   folderPath: none
   filePath: pcd_models/IkeaRedBowl/IkeaRedBowl.pcd
   fromPackage: true
   packageName: rs_resources
   scale: 100.0
   axisMap: [-1,2,-3]
Mug:
   category: Mug
   color: [red, Red]
   shape: [round,Cylindrical]
   material: [Plastic]
   openability: false
   folderPath: none
   filePath: pcd_models/IkeaRedCup/IkeaRedCup.pcd
   fromPackage: true
   packageName: rs_resources
   scale: 100.0
   axisMap: [3,1,2]
#Spoon:
#   category: BluePlasticSpoon
#   color: [blue,Blue]
#   shape: [flat, Filiform]
#   material: [Plastic]
#   openability: false
#   folderPath: none
#   filePath: pcd_models/BluePlasticSpoon/BluePlasticSpoon.pcd
#   fromPackage: true
#   packageName: rs_resources
#   scale: 50.0
#   axisMap: [-3,-1,2]
#Bowl:
#   category: EdekaRedBowl
#   color: [white,red,White,Red]
#   shape: [round,Conical]
#   material: [Plastic]
#   openability: false
#   folderPath: none
#   filePath: pcd_models/EdekaRedBowl/EdekaRedBowl.pcd
#   fromPackage: true
#   packageName: rs_resources
#   scale: 80.0
#   axisMap: [3,1,2]
#Mug:
#   category: CupEcoOrange
#   color: [yellow,Brown]
#   shape: [round,Cylindrical]
#   material: [Plastic]
#   openability: false
#   folderPath: none
#   filePath: pcd_models/CupEcoOrange/CupEcoOrange.pcd
#   fromPackage: true
#   packageName: rs_resources
#   scale: 100.0
#   axisMap: [1,2,3]
